client/src/components/body/profilepage/Profile.js

```diff
@@ -49,34 +48,38 @@ const Profile = () => {
   }
 
   return (
-    <React.Fragment>
+    <div className={"tw-mt-[3rem]"}>
```

Comment:
This is my personal opinion, I think there needs to be bigger margins to the left and right for the entire page.
![Screenshot 2024-10-25 at 1 12 06 PM](https://github.com/user-attachments/assets/039d2f93-c7c2-475d-b330-fae107b86af1)
Example, the "Have a group code?" section seems very close to the edge of the screen

Reply:
![Screenshot 2024-10-25 at 1 12 46 PM](https://github.com/user-attachments/assets/9f40e3b8-58ba-4692-98d0-2bc94ca01b74)
Same with the "View Your Instructor Groups" section. Also, the accessibility buttons should not be touching the edge of the screen